packages/connect-node-test/src/helpers/test-routes.ts

```diff
@@ -57,8 +57,10 @@ const testService: ServiceImpl<typeof TestService> = {
     ]);
   },
 
-  cacheableUnaryCall(/*request*/) {
-    throw new ConnectError("TODO", Code.Unimplemented);
+  cacheableUnaryCall(request, context) {
+    // TODO: Need to be able to determine request protocol/HTTP method.
+    context.responseHeader.set("request-protocol", "unknown");
```

Comment:
We can add two properties to HandlerContext:

```ts
  readonly protocolName: string;
  readonly requestMethod: string;
```

Both values are available when we call createHandlerContext() in the handler-factory.ts files. We should wait for consensus on https://github.com/bufbuild/connect-go/pull/502.

Reply:
Do you think we should block this PR on resolving this TODO? I'd like to get consensus on the other issue, but not 100% sure how long it will take either. It's not a huge deal though, since this isn't a blocker for the client part, which I think is more important anyways.

Reply:
Since we have consensus on connect-go now, I went ahead and implemented this as-planned.